### DISCUSSION TITLE
Feedback on code font fallback to Courier New

### DISCUSSION THREAD

**Discussion by dprothero** [twilio-labs/paste#2345] [Ideas] [answered]
Small nit... `font-family-code` is `'Fira Mono', 'Courier New', Courier, monospace`, but, when falling back to Courier New, it's so light, it's hard to read.

![image](https://user-images.githubusercontent.com/1897006/162068177-7dbd2bd6-ca45-41d5-8873-2dc88ddeb629.png)

just plain `Courier` is a much easier to read font

![image](https://user-images.githubusercontent.com/1897006/162068357-312d557c-6abe-4501-9dc9-d0fc752c6b82.png)

**Comment by TheSisb** [comment]
Hi @dprothero,

Thanks for reporting this! We're still discussing internally and will report back when we have settled on a plan of action. 

Out of curiosity, how is `Fira Mono` falling through for you? It should be included in [the CDN hosted fonts file](https://assets.twilio.com/public_assets/paste-fonts/main-1.2.0/fonts.css) we [recommend adding to your page's `<head>` tag](https://paste.twilio.design/introduction/for-engineers/manual-installation#how-to-load-the-right-font).

**Reply by dprothero** [reply]
I hadn't added the reference to the fonts file, yet, which is how I noticed it. So, yeah, small issue since most folks will see the `Fira Mono` font.

**Reply by TheSisb** [reply]
I've made a ticket to update the fallback per your suggestion. Thank you for sharing that context and for helping make Paste better!

**Comment by TheSisb** [comment, marked as answer]
I just merged a PR with the suggested change: https://github.com/twilio-labs/paste/pull/2355. This will be released sometime over the next week.

**Reply by dprothero** [reply]
Nice!

**Reply by gloriliale** [reply]
This has just been released! 🎉
